.github/CONTRIBUTING.md

```diff
@@ -93,22 +93,19 @@ Unsure where to begin contributing? You can start by looking through these good
 * Do not include issue numbers in the first line of your commit message
 * Reference issues and pull requests liberally after the first line
 * [Verify your commits][signing-commits] before creating a pull request
-* Standardize your commits:
-    - use `yarn commit` to write the commit message with the interactive guide
-    - clean up the problem/warnings from check before pushing your commits
+* Standardize your commits
```

Comment:
I think we should remind the contributors what convention we actually use (changelog I think)

Reply:
That is explained in STYLEGUIDE.md, but I think it is a good idea to write an example of how to write a commit message:
```
feat(ui-compo): implement button component


test(button): fix button behavior
```

Reply:
Cool!